www/src/Data/Pins.ts

```diff
@@ -57,6 +57,8 @@ export const BUTTON_ACTIONS = {
 	BUTTON_PRESS_E10: 52,
 	BUTTON_PRESS_E11: 53,
 	BUTTON_PRESS_E12: 54,
+  BUTTON_PRESS_TJ_TOGGLE_1: 55,
```

Comment:
@tupleu I'm sorry for being some random GitHub user giving you this feedback, but it looks like this line has spaces instead of tabs.

P.S. I'd like to contribute to this repository in the future, so I was just browsing some pull requests to get familiarized with the code.

Reply:
Thank you for the feedback. Nice catch, I'll fix it in the next commit.